schemas/2015-10-01/Microsoft.DataFactory.json

```diff
@@ -0,0 +1,203 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2015-10-01/Microsoft.DataFactory.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.DataFactory",
+    "description": "Microsoft Azure Data Factory Resource Types",
+    "resourceDefinitions": {
+        "dataFactories": {
+            "description":  "The top level DataPipeline resource container.",
+            "allOf": [
+                {
+                    "required": [
+                        "type"
+                    ],
+                    "properties": {
+                        "type": {
+                            "enum": [ "Microsoft.DataFactory/datafactories" ]
+                        }
+                    }
+                },
+                {
+                    "$ref": "#/definitions/apiVersionCommon"
+                }
+            ]
+        },
+        "linkedServices": {
+            "description": "Defines the information needed for Data Factory to connect to external resources.",
+            "$ref": "#/definitions/linkedServiceBase"
+        },
+        "datasets": {
+            "description": "A named reference/pointer to the data to be used as an input or an output of an Activity. Datasets identify data structures within different data stores including tables, files, folders and documents.",
+            "$ref": "#/definitions/datasetBase"
+        },
+        "pipelines": {
+            "description": "A logical grouping of Activities. that operate as a unit that together perform a task.",
+            "$ref": "#/definitions/pipelineBase"
+        }
+    },
+    "definitions": {
+        "apiVersionCommon": {
+            "required": [
+                "apiVersion"
+            ],
+            "properties": {
+                "apiVersion": {
+                    "enum": [
+                        "2015-10-01"
+                    ]
+                }
+            }
+        },
+        "datafactoryCommon": {

```

Comment:
I see, the diff did not make that obvious. I made another mistake here, it should be referenced on line 21 instead of apiVersionCommon.